src/commands/purge.ts

```diff
@@ -10,7 +10,11 @@ export default Purge = class {
     return 'Purges the channel it is called within. Restricted to Board Members and Administrators.';
   }
 
-  public static execute(args: string[], msg: Message, bot: Client) {
+  public static execute(
+    args: string[],
+    msg: Message,
+    { client: bot }: { client: Client }
+  ) {
```

Comment:
The rest of this file has a high cognitive complexity (rated 9 by code climate out of a max of 5).
After these tests are in place.

Opened #110